### DISCUSSION TITLE
Export folder not mounted and can't find path (CommandError: That path doesn't exist)

### DISCUSSION THREAD

**Discussion by freaky33** [jonaswinkler/paperless-ng#831] [Support]
I'm trying to export my documents to an external drive. Event though the path definitely exists, I get the error

`CommandError: That path doesn't exist`

Also I can't find data exported to the home directory /paperless-ng/export in the /export folder in my home system as described in the docs. What am I doing wrong?

**Comment by jonaswinkler** [comment]
If you're using docker, the exporter exports to a folder *inside* the docker container. If you specify "../export" as stated in the documentation, the export lands on a volume that's bind mounted to a folder on your host system.

Also, the issue tracker is for actual bugs.

**Reply by freaky33** [reply]
I'm sorry for using the wrong category. 

I know, I've tried using ../export. I might have misunterstood. So if I installed it using docker in the home directory of a dedicated user it will be /home/***user***/paperless-ng/export? 

Thanks and sorry for the inconvience
